module-4/backend/week_one.md

```diff
@@ -5,17 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+How functions and loops work in JS.
+
 2. What are some tools to help debug JavaScript code?
+debugger, console.log, alert, developer tools
+
 3. What are some tools you need in order to unit test your JavaScript?
+Installing and requiring the different libraries. Chai as an example and requiring the commands (expect, assert) that you would want to use.
+
 4. What is the syntax for invoking a function?
+functionName();
+//If function takes in an arguement then you pass in inside the paras.
+
 5. What is CORS?
+Cross Origin Resource Sharing. Seems to be the protocol for how one app controls its assets from another app, allowing it to access that resource or not? 
+
 6. How do we enable CORS?
+Depending on the framework being used, there exists a specific command/file you utilize?
+
 7. What's `this` in JavaScript?
+It's a special reserved term. References it's parent object and is dependent on the context of where it's referenced.
+
 8. What is Webpack and why is it useful?
+Similiar to the Asset Pipeline in Rails, it compiles everything and renders a file that the browser takes in.
+
 9. When/why do you want to use event delegation?
+Deals with event bubbling. If you set an event listener on a parent object, the child elements will then inherit that functionality.
```

Comment:
Especially when wanting to listen for events on elements that have not yet been loaded into the DOM :)